.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
+}
```

Comment:
Don't add unrelated things.

Reply:
Oops, had to ignore the file. Now you know why the formatting issues keep hunting me ;)